dpnp/random/_random.pyx

```diff
@@ -48,6 +48,17 @@ ctypedef void(*fptr_mkl_rng_gaussian_1out_t)(void *, size_t)
 ctypedef void(*fptr_mkl_rng_uniform_1out_t)(void *, long, long, size_t)
 
 
+cpdef dpnp_srand(seed=None):
+    """
+    Initialize basic random number generator.
+
+    """
+    if seed is None:
+        dpnp_srand_c()
```

Comment:
I would propose to use one function and use some `std::time(0)` as initializer at python level